models/agent_model.go

```diff
@@ -405,6 +419,20 @@ func (s *Agent) DSN(service *Service, dialTimeout time.Duration, database string
 			} else {
 				sslmode = "verify-full"
 			}
+			if s.PostgreSQLOptions != nil {
+				if files := s.Files(); len(files) > 0 {
+					for key := range files {
+						switch key {
+						case "tlsCa":
+							q.Set(key, tdp.Left+" .TextFiles.tlsCa "+tdp.Right)
+						case "tlsCert":
+							q.Set(key, tdp.Left+" .TextFiles.tlsCert "+tdp.Right)
+						case "tlsKey":
+							q.Set(key, tdp.Left+" .TextFiles.tlsKey "+tdp.Right)
```

Comment:
as far as I see `s.Files()` returns
```
map[string]string{
				"ssl_ca_file":   s.PostgreSQLOptions.TLSCa,
				"ssl_cert_file": s.PostgreSQLOptions.TLSCert,
				"ssl_key_file":  s.PostgreSQLOptions.TLSKey,
			}
```
so this code will be skipped.
Please add more tests.